package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Cap-BnB",
+  "version": "",
+  "description": "",
+  "author": "Landon Metcalf",
+  "license": "",
+  "main": "server/server.js",
+  "scripts": {
+    "test": "jest",
+    "start": "nodemon server.js",
+    "react-dev": "webpack -d --watch"
+  },
+  "dependencies": {
+    "@shelf/jest-mongodb": "^1.1.3",
+    "axios": "^0.19.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.7.12",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "react-icons": "^3.8.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.7.0",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-react": "^7.7.0",
+    "babel-loader": "^8.0.6",
+    "chai": "^4.2.0",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```

Comment:
Remove this

Reply:
@FrancoisArago - remove what exactly? All 4 lines that correspond to this comment or just the es-lint line, 29?

Reply:
`eslint-config-hackreactor` - sorry if multiline comment occurred - my mistake!